eZ/Publish/Core/FieldType/RichText/Resources/stylesheets/ezxml/docbook/core.xsl

```diff
@@ -351,9 +356,7 @@
           <xsl:value-of select="@class"/>
         </xsl:attribute>
       </xsl:if>
-      <xsl:element name="para" namespace="http://docbook.org/ns/docbook">
-        <xsl:apply-templates/>
-      </xsl:element>
```

Comment:
Could you elaborate on results of that fix? What kind of transformation (input XML, output XML) does this fix?
I'm not that fluent in XSLT to get it by just looking at the markup, however this worries me a bit, because `<para>` is something that we use.

Reply:
I have a explanation of what is going on in the description of the [related PR](https://github.com/ezsystems/ezplatform-xmltext-fieldtype/pull/42) in ezplatform-xmltext-fieldtype. Is that one ok, or you need more details?